sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -472,8 +472,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [python-fr-product-docs]: https://docs.microsoft.com/azure/cognitive-services/form-recognizer/overview
 [python-fr-ref-docs]: https://aka.ms/azsdk/python/formrecognizer/docs
 [python-fr-samples]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer/samples
-[train-a-model-using-labeled-data]: https://docs.microsoft.com/azure/cognitive-services/form-recognizer/quickstarts/python-labeled-data#train-a-model-using-labeled-data
-
+[train-a-model-using-labeled-data]: https://docs.microsoft.com/azure/cognitive-services/form-recognizer/quickstarts/get-started-with-form-recognizer#train--analyze-a-custom-form
```

Comment:
This link seems to be more generic than the previous one, is that what the change is meant for? Or should it be a translation to the newer link for the python specific example: `https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/client-library?tabs=2-1&pivots=programming-language-python#train-a-custom-model` ?

Reply:
I clicked on the link you put and it shows how to label with the interface they provide, just bringing it up to see if that's better than the other link. The article this points to starts with: `Get started with the Form Recognizer using the Form Recognizer Sample Tool.`

Reply:
The old link doesn't "exist" anymore on MS docs - it's pointing to an old archive of it on Github. The link is intended to be a conceptual instruction on setting up training. That being said, I removed it from the README because it's actually not being used.

Reply:
Sorry I knew I did not write my comment clearly. I totally get that the old link doesnt exist anymore, but it seems it was updated to this one: https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/client-library?tabs=2-1&pivots=programming-language-python#train-a-custom-model , right?
My main question was where we're updating this same link should we update it to the new version (of its old self 😝)?

Reply:
I understood. This link is supposed to be an explanation of setting up training - setting up the storage container, getting the SAS URL, possibly labeling the files. Service concepts, not SDK sample. The old link is the best we had before they changed it - and also didn't include any SDK code.

Reply:
Ok gotcha. I didnt like the new link as much since it is more focused towards the sample tool, but I agree it has some more information than the Python specific one.